WcaOnRails/app/views/competitions/_competition_form.html.erb

```diff
@@ -229,41 +229,46 @@
-    <% end %>
-
-    <% disable_money_input = !@competition.can_edit_registration_fees? %>
-    <%= f.input :currency_code, collection: Money::Currency.table.values, label_method: lambda { |c| c[:name]  }, value_method: lambda { |c| c[:iso_code] }, disabled: disable_money_input  %>
-    <%= f.input :base_entry_fee_lowest_denomination, as: :money_amount, currency: @competition.currency_code, disabled: disable_money_input, currency_selector: "#competition_currency_code" %>
-
-    <%= f.input :use_wca_registration %>
+      <%= f.input :use_wca_registration %>
 
-    <div class="wca-registration-options">
-      <%= f.input :enable_donations %>
-
-      <%= f.input :guests_enabled, as: :radio_buttons, collection: [ :true, :false ] %>
+      <div class="wca-registration-options">
+        <% if @competition.can_receive_registration_emails?(current_user.id) %>
+          <%# Quick hack to fill receive_registration_emails with the correct value for the current_user %>
+          <% @competition.receive_registration_emails = @competition.receiving_registration_emails?(current_user.id) %>
+          <%= f.input :receive_registration_emails, as: :boolean %>
+        <% end %>
+        <%= f.input :enable_donations %>
+        <%= f.input :guests_enabled, as: :radio_buttons, collection: [ :true, :false ] %>
+      </div>
 
-      <% if @competition.can_receive_registration_emails?(current_user.id) %>
-        <%# Quick hack to fill receive_registration_emails with the correct
-            value for the current_user %>
-        <% @competition.receive_registration_emails = @competition.receiving_registration_emails?(current_user.id) %>
-        <%= f.input :receive_registration_emails, as: :boolean %>
-      <% end %>
+      <%= f.input :guests_entry_fee, as: :money_amount, currency: @competition.currency_code, currency_selector: "#competition_currency_code" %>
 
       <div class="col-sm-offset-2 col-sm-9">
         <div class="row datetimerange">
           <%= f.input :registration_open, as: :datetime_picker, wrapper: :ranged_datetime %>
           <%= f.input :registration_close, as: :datetime_picker, wrapper: :ranged_datetime %>
         </div>
       </div>
-    </div>
 
-    <% if @competition_admin_view || !is_actually_confirmed %>
       <%= f.input :competitor_limit_enabled %>
       <div class="wca-competitor-limit-options">
         <%= f.input :competitor_limit %>
         <%= f.input :competitor_limit_reason %>
       </div>
+
+      <% disable_money_input = !@competition.can_edit_registration_fees? %>
+      <%= f.input :currency_code, collection: Money::Currency.table.values, label_method: lambda { |c| c[:name]  }, value_method: lambda { |c| c[:iso_code] }, disabled: disable_money_input  %>
+      <%= f.input :base_entry_fee_lowest_denomination, as: :money_amount, currency: @competition.currency_code, disabled: disable_money_input, currency_selector: "#competition_currency_code" %>
+
+      <%= f.input :refund_policy_amount %>
```

Comment:
I've been looking how to display this with the "%" symbol in the form, just as it's done in the public pages, but I didn't find anything. Anyone knows how to do it?

Reply:
Sorry, I'm not sure I understand what you're asking for here. What public pages are you referring to?

Reply:
I'm referring to info and register, where the registration requirements are shown.

Reply:
Oh, are you referring to where you use the `number_to_percentage` helper? That's just static content, though. AFAIK, we don't have any sort of input element anywhere on the WCA website where we ask the user for a percentage. You could imagine building your own using bootstrap's input groups. See https://getbootstrap.com/docs/3.3/components/#input-groups, and here's a simple example I put together: http://jsfiddle.net/S9qrZ/1783/.

To do this cleanly, we'd could create a new type of input element over in WcaOnRails/app/inputs. However, this review has already got a lot of moving pieces, so I think it would be best to take that on in a later PR. It's getting hard for me to follow all of this.

This reminds me that I think it would be good to rename "refund_policy_amount" to something like "refund_policy_percent".

Reply:
Ok, I can leave it like it is and open an issue to track the input as percent. I will change `refund_policy_amount` to `refund_policy_percentage` if that's ok and make a last commit to finish with this PR.